ExtIO_sddc/ExtIO_sddc.cpp

```diff
@@ -668,24 +671,23 @@ int EXTIO_API ExtIoGetSrates(int srate_idx, double * samplerate)
 	*samplerate = 1000000.0 * (div * 2);
 	if (*samplerate / RadioHandler.getSampleRate() * 2.0 > 1.1)
 		return -1;
-     DbgPrintf("*ExtIoGetSrate idx %d  %e\n", srate_idx, *samplerate);
+	DbgPrintf("*ExtIoGetSrate idx %d  %e\n", srate_idx, *samplerate);
 	return 0;
 }
 
 extern "C"
 int  EXTIO_API ExtIoGetActualSrateIdx(void)
 {
-    EnterFunction();
+	EnterFunction();
 	if (RadioHandler.GetmodeRF() == VHFMODE)
 		return giExtSrateIdxVHF;
 	else
 		return giExtSrateIdxHF;
 }
 
-extern "C"
-int  EXTIO_API ExtIoSetSrate(int srate_idx)
+static int ExtIoSetSrateImpl(int srate_idx, bool internal_call)
```

Comment:
I prefer naming it as SetSrateInternal. Extio prefix should use when it is part of extio contract